### DISCUSSION TITLE
Is it possible to wrap Latex blocks into a div, like it's done with code?

### DISCUSSION THREAD

**Discussion by queirozfcom** [quarto-dev/quarto-cli#6018] [Q&A] [answered]
### Description

Latex Blocks **overflow** and break the HTML page's style when the viewport is small (using `MathJax` renderer)

![xx](https://github.com/quarto-dev/quarto-cli/assets/1554354/8f944141-cee2-467d-96a0-330a9bf1c862)

This is how I'm writing this Latex block in the `qmd` file:

```
$$
Output_{Simplest\ Possible\ LM} = \frac{\#\ valid\ words\ in\ input\ sequence}{\#total\ words\ in\ input\ sequence}
$$
```

Is it possible to wrap the Latex cell inside a div as we have with code blocks? (**example below**)

![xx](https://github.com/quarto-dev/quarto-cli/assets/1554354/41219299-763c-42f7-8192-e658fff580e5)

**Comment by queirozfcom** [comment]
I've already tried adding overflow-x: scroll to all sorts of CSS selectors:

```
mjx-container, mjx-math, mjx-mtable, mjx-table, mjx-itable, .MathJax_Display, .MathJax, .MJXc-display, .MathJax_SVG_Display {
    overflow-x: scroll;
}
```

**Reply by baptiste** [reply]
This seems to work though

````
---
title: "Untitled"
format: 
  html:
    max-width: 200px
    include-in-header: 
       - text: |
          <style>
            .MathJax {
             overflow-x: auto;
             overflow-y: hidden;
            }
          </style>
---

$$
\alpha = \frac{\text{this is verrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrry long}}{\text{indeed}}
$$

````

**Reply by queirozfcom** [reply]
For me, it doesn't =/

You can see the CSS style is being picked up

![xx](https://github.com/quarto-dev/quarto-cli/assets/1554354/76941b71-c1c6-4d92-af5f-d6316ef92278)

**Reply by queirozfcom** [reply]
I event tried the good ole `!important`

![xx](https://github.com/quarto-dev/quarto-cli/assets/1554354/967162a3-900e-4a6b-acca-1c0bd61a90fb)

**Comment by queirozfcom** [comment]
What I ended up doing (not as pretty as mathjax but at least it doesn't mess up the scrolling bars)

- using `webtex` instead of `mathjax`
- using this CSS code:
    ```   
    .math.display {
     max-width: 100%;
    }
    ``` 
OUTPUT: 
![xx](https://github.com/quarto-dev/quarto-cli/assets/1554354/84e1867d-7949-42ba-9748-a9aee424df53)

**Reply by mcanouil** [reply]
It would help to specifically target your use case if you could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

Also, while screenshots can be useful, without alternative texts and no code users with impaired vision won't be able to use/understand this thread.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by mcanouil** [comment, marked as answer]
Some related solutions:

- https://github.com/quarto-dev/quarto-cli/discussions/2170#discussioncomment-3502467
- https://github.com/quarto-dev/quarto-cli/discussions/2170#discussioncomment-3502760

Additionally, you could tweak MathJax config:

```
MathJax.Hub.Config({
  MathZoom: {
    styles: {
      "#MathJax_Zoom": {
        overflow: "auto ! important"
      }
    }
  }
});
```

**Reply by queirozfcom** [reply]
Thank you very much @mcanouil 

[This answer](https://github.com/quarto-dev/quarto-cli/discussions/2170#discussioncomment-3502760) did the trick (enclosing the `$$` block with triple colons (`:::`) and adding an inline CSS style
